docs/README.md

```diff
@@ -94,7 +93,7 @@ options:
   --profile USER_CONFIG_PROFILE
                         Tokendito configuration profile to use.
   --config-file USER_CONFIG_FILE
-                        Use an alternative configuration file. Defaults to ~/.local/share/tokendito/tokendito.ini 
+                        Use an alternative configuration file. Defaults to tokendito.ini with location depending on the OS.
```

Comment:
I think we may want to have some examples about where the config file can be found in the 3 supported operating systems?

Reply:
There is a chance that it changes in the future, perharps adding a to platformdirs?